### DISCUSSION TITLE
stoicism/wardrobe/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#11] [General]
# stoicism/wardrobe/

https://shanenull.com/stoicism/wardrobe/

**Comment by shane0** [comment]
# favorites

- started an album here https://shanenull.com/photography/wardrobe/

**Reply by shane0** [reply]
https://www.knickerbocker.nyc/products/gym-sweatpant

**Reply by shane0** [reply]
https://dehen1920.com/collections/jackets/products/miller-coat-pre-order?variant=45686162292962
 
Abraham Moon & Sons
 
Halley Stevensons evermax

**Reply by shane0** [reply]
Abraham Moon® Shetland Plain Twill (UK)
 100% British Shetland Wool

**Reply by shane0** [reply]
14 oz. Fox Brothers® Harvest Houndstooth Check Twill Flannel Pocketing (UK)

**Comment by shane0** [comment]
This Sukajan Souvenir jacket from Pherrow's comes cut in a regular fit and features embroidery citing Ukiyo-e and Japan's Edo period. The embroidery on this jacket depicts a school of koi fish, a common illustration in Japan representing luck, prosperity, and longevity. The reverse side features a flock of roosters mimicking the "Ito Wakaoki" style of Ukiyo-e. The jacket comes finished with ribbed detailing, pockets at the waistband, and Japanese YKK zipper.

**Reply by shane0** [reply]
uwabaki house shoes

**Comment by shane0** [comment]
kasuri chambray gabardine

**Comment by shane0** [comment]
Red cloud
Always be prepared for good times

**Comment by shane0** [comment]
https://stagprovisions.com/products/lamar-jacquard-woven-blanket-stripe-shirt-nutshell

**Comment by shane0** [comment]
https://www.bergdorfgoodman.com/p/kardo-men-s-corduroy-block-print-casual-shirt-prod183200042?utm_source=google_shopping&ecid=BGCS_GP_CV_NC_PR_NONE&gad_source=1&gclsrc=ds

red cloud dyneema pants

**Reply by shane0** [reply]
https://redcloudscollective.com/products/gn-01-fitted-work-pant-dyneema-denim

**Reply by shane0** [reply]
https://www.theshopkeeperstore.com/products/chintan-short-sleeve-shirt-leaf-block-print-1

**Reply by shane0** [reply]
https://www.nordstrom.com/s/whiting-mercer-check-button-up-overshirt/7687274

**Comment by shane0** [comment]
# considering

https://merzbschwanen.com/shop/2s18-507-2s18-men-s-loopwheeled-long-sleeve-heavy-13-4oz-relaxed-fit-507-ruby-red-11208?collection=1&product=product.template%2811208%2C%29

**Reply by shane0** [reply]
Pure Blue Japan KS-5011-SS Kasuri T-shirt Black

**Reply by shane0** [reply]
https://cumulus.equipment/us_en/men-jacket-climalite-vest.html

**Comment by shane0** [comment]
# dehen 1920

**Reply by shane0** [reply]
https://poshmark.com/search?query=dehen%201920&department=Men&my_size=true

**Reply by shane0** [reply]
- for my birthday & found at discount
- [raincoat](https://dehen1920.com/collections/jackets/products/heavy-duty-raincoat-nautical-red)
- going to have to find an adventure worthy of this thing
- maybe an old school mock hunt somewhere very cold 
- with just scouting or shedding unless I actually find a species in actual overshoot
- no backpack just use all of these giant pockets
- layers: long teeshirt + dehen moto hoode + ginew lined vest + raincoat I should be invincible
- experience respect for the old ways when people hand made everything and cared about the function of gear & provisions
-  rewax it
- find a horse and go get lose & sleep in a canvas tent & cowboy roll

**Reply by shane0** [reply]
melton wool
worsted wool
four-end jersey knit

**Reply by shane0** [reply]
Celebrating Max’s first pct section hike Nihon Menpu
 11oz broken twill
https://dehen1920.com/products/crissman-overshirt-indigo-denim

**Comment by shane0** [comment]
# boston

- topdrawer & boston general store
- kolo
- Nishiguchi Kutsushita's Linen Cotton Anklet 
- travelers notebook
- helvetica pencils
- pinder bros flask

**Reply by shane0** [reply]
Nishiguchi Kutsushita's Linen Cotton Anklet

**Reply by shane0** [reply]
NWOT Dehen 1920 Heritage Men’s Gray Wool Shawl Sweater Coat Cardigan MIUSA L

**Reply by shane0** [reply]
Nishiguchi Kutsushita's Linen Cotton Anklet wears out easily

**Comment by shane0** [comment]
# mountains

- day Tilley hat stone glacier pants shirt vest rab raincoat
- night alpha hoodie apex hoody boots sleeping bag

**Comment by shane0** [comment]
# cowl

- knit cowl?

"coif" or "chaperon"
bourrelet or roundlet


Here are terms from various languages for **hood-like garments that cover the head and shoulders**, often as standalone pieces or part of traditional attire:

---

### **1. **Burnous** (North Africa / Arabic: بُرْنُوس‎)**  
- A **hooded cloak** made of wool, traditionally worn by Berber and Arab communities.  
- Covers the head, shoulders, and upper body.  

---

### **2. **Capuchón** (Spanish)**  
- A **large hood** attached to a cloak or robe, historically worn in Europe (similar to the **cowl**).  
- Associated with monastic robes (e.g., Capuchin monks).  

---

### **3. **Gugel** (Medieval German)**  
- A **hood with a trailing tail** that could be wrapped around the neck or shoulders.  
- Evolved into a fashion item for warmth and status.  

---

### **4. **Chador** (Persian: چادر)**  
- A full-body cloak worn by some Iranian women, often with a **hood-like head covering** that drapes over the shoulders.  

---

### **5. **Amauti** (Inuit)**  
- A traditional Inuit **parka with a large hood** designed to carry a baby, covering the shoulders and upper back.  

---

### **6. **Cappa** (Italian)**  
- A medieval **hooded cape** that covers the shoulders, often worn by travelers or clergy.  

---

### **7. **Salakot** (Philippines)**  
- A wide-brimmed hat, but some versions include **attached cloth hoods** that drape over the shoulders for rain protection.  

---

### **8. **Hooded Abaya** (Arabic: عباءة)**  
- A long, hooded robe worn in some Middle Eastern cultures, covering the head and shoulders.  

---

### **9. **Tudung** (Malay/Indonesian)**  
- A headscarf that sometimes extends to cover the shoulders, worn in Southeast Asia.  

---

### **10. **Ghoonghat** (Hindi: घूँघट)**  
- A veil or hood-like drape from South Asia, traditionally covering the head and shoulders.  

---

### **Summary Table**  
| Term              | Origin/Culture       | Description                                  |  
|--------------------|----------------------|----------------------------------------------|  
| **Burnous**        | North Africa         | Hooded wool cloak for head/shoulders.        |  
| **Capuchón**       | Spanish/European     | Monastic hooded robe with shoulder coverage. |  
| **Gugel**          | Medieval German      | Hood with a tail for neck/shoulder warmth.   |  
| **Amauti**         | Inuit                | Parka hood for carrying babies, covers back. |  
| **Chador**         | Persian/Iranian      | Full-body cloak with a hooded headpiece.     |  

---

### **Modern Equivalents**  
- Terms like **"hooded shawl"** or **"hooded cape"** are used in English, but they lack the cultural specificity of the above terms.  

These garments often blend practicality (weather protection) with cultural or religious symbolism. Let me know if you’d like more examples! 🧣🌍
